flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Refuel.js

```diff
@@ -163,5 +163,8 @@ class A32NX_Refuel {
                 return;
             }
         }
+
+        // DONE FUELING
+        SimVar.SetSimVarValue("L:A32NX_REFUEL_STARTED_BY_USR", "Bool", false);
```

Comment:
Not a good idea to write this every update if it's already false (writing local vars is a little costly).

Reply:
We check whether this is set to false and return early at the start of the update method, so this should be fine, no?

Reply:
Ah, my bad, you are correct. 👍